build/win32/Cargo.toml

```diff
@@ -1,19 +0,0 @@
-[package]
```

Comment:
mistake?

Reply:
hah, looks like a merge oopsie. This should be fixed in a separate PR though, could you handle cleaning that up?

Reply:
Can do.